environs/networking.go

```diff
@@ -42,6 +42,18 @@ type Networking interface {
 	// provider that have subnets available.
 	Spaces() ([]network.SpaceInfo, error)
 
+	// ProviderSpaceInfo returns the details of the space requested as
+	// a ProviderSpaceInfo. This will contain everything needed to
+	// decide whether an Environ of the same type in another
+	// controller could route to the space. If "" is passed as
```

Comment:
Can we define a const DefaultSpace or something for ""
Or maybe we have one. But if not, we should.